### PR TITLE
🖍 [amp-story-player] Extract iframe css to different file

### PR DESCRIPTION
**Pull request by Enriqe**
* Create a separate css file for iframe as suggested in https://github.com/ampproject/amphtml/pull/26557#discussion_r373217335 by @alanorozco
* Updates OWNERS so that we have write permissions for `amp-story-player-*.css` files.

Partial for #24539 #26308

### CODE REVIEW

File: build-system/tasks/css.js [hunk 1]
```diff
@@ -59,6 +59,11 @@ const cssEntryPoints = [
     outJs: 'amp-story-player.css.js',
     outCss: 'amp-story-player-v0.css',
   },
+  {
+    path: 'amp-story-player-iframe.css',
+    outJs: 'amp-story-player-iframe.css.js',
+    outCss: 'amp-story-player-iframe-v0.css',
```
Comment (gmajoulet) [inline build-system/tasks/css.js]: Does this match the name of the JavaScript bundle?

Comment (gmajoulet) [reply build-system/tasks/css.js]: Actually nvm, I got confused with the other CSS file that publishers have to import.

File: css/OWNERS [hunk 2]
```diff
@@ -10,5 +10,9 @@
         {name: 'camelburrito', requestReviews: false},
       ],
     },
+    {
+      pattern: '{amp-story-player,amp-story-player-iframe}.css',
```
Comment (alanorozco) [inline css/OWNERS]: nit: could this be changed to only `amp-story-*`?

Comment (alanorozco) [reply css/OWNERS]: and maybe set to 'ampproject/wg-stories' rather than a static set?

Comment (Enriqe) [reply css/OWNERS]: We think wg-stories is too wide of a group, so we prefer having a static set of usernames for now.

File: build-system/tasks/css.js (hunk 1 above)
Comment (newmuis) [reply build-system/tasks/css.js]: I also got confused by this; should we add comment(s) distinguishing them?

Comment (Enriqe) [reply build-system/tasks/css.js]: Done.

**Comment by Enriqe** [comment]
Thanks all!

**Pull request merged**